src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/pdl/PdlPersonUtil.kt

```diff
@@ -24,6 +24,8 @@ fun List<Oppholdsadresse>.gjeldende(): Oppholdsadresse? = this.find { !it.metada
 fun List<Sivilstand>.gjeldende(): Sivilstand = this.find { !it.metadata.historisk } ?: this.first()
 fun List<Fødsel>.gjeldende(): Fødsel = this.first()
 fun List<DeltBosted>.gjeldende(): DeltBosted? = this.find { !it.metadata.historisk }
+fun List<DeltBosted>.gjeldendeListe(): List<DeltBosted> = this.filter { !it.metadata.historisk }
```

Comment:
Litt usikker på dette med historisk. Med tanke på hvordan allt annet er her, er det kun sånn at man kun kan ha 1 gjeldende delt bosted?

Dvs kun 1 med historisk = false? 
 
 
`gjeldende` på DeltBosted er ikke i bruk, så burde kunne erstatte den, samtidig som jeg er usikker på om dette er riktig.

Reply:
Her nevnes det at det kun finnes 1 gjeldende
https://nav-it.slack.com/archives/C84H68ESC/p1652872685923589?thread_ts=1652871934.730029&cid=C84H68ESC

gjeldende = historisk == false https://pdldocs-navno.msappproxy.net/ekstern/index.html#_funksjonell_historikk

Dvs skal kun finnes en gjeldende?

Reply:
Ja du har rett. Jeg har snakket med Hanne og sett litt på kontrakten i bufdir. Virker som om det bare er en "ikke historisk"